### PR TITLE
changed banner text color to white

### PR DESCRIPTION
**Pull request by manavsiddharthgupta**
issue - Banner text color wrong for https://k8s.io/partners/ 
changes - text color black to white

### CODE REVIEW

**Comment by sftim** [comment]
@manavsiddharthgupta this PR actually leaves the banner text color to be inherited

File: static/css/gridpage.css [hunk 1]
```diff
@@ -33,7 +33,7 @@
 }
 
 .gridPage p {
-    color: rgb(26,26,26);
+    color: rgb(255, 255, 255);
```
Comment (sftim) [inline static/css/gridpage.css:36]: What happens if we wholly omit setting a color here?

**Comment by Sea-n** [comment]
Hey @manavsiddharthgupta, thanks for your contribution! Are you ready to do some follow-up on this PR? 😊

**Comment by sftim** [comment]
My worry: some other change combines with this to put page text in that is white-on-white, and hence invisible.

**Review by divya-mohan0209 — COMMENTED** [review]
Hello @manavsiddharthgupta, please could you advise if you're ready to address the feedback provided by @sftim?

**Comment by kbhawkey** [comment]
👋 @manavsiddharthgupta . I'm placing a hold on this pull request.
You can `/hold cancel` at any time.
/hold

**Comment by Sea-n** [comment]
Hey @manavsiddharthgupta,

Thanks for purposing this PR. But since you didn't respond for a while, we have to close this PR now.
/close

Feel free to `/reopen` when you are ready to move forward.

**Pull request closed without merging**